README.md

```diff
@@ -130,14 +130,25 @@ interesting part of this framework:
   the methods to reload a saved experiment, or to plot the powergrid given an observation for
   example. This is an introductory notebook. More user friendly graphical interface should
   come soon.
-
+* [5_RedispathingAgent](getting_started/5_RedispathingAgent.ipynb) explains what is the "redispatching" from a
```

Comment:
explains what is the "redispatching" from the point of view of a 
  company who's in charge of keeping the powergrid safe (aka a Transmission System Operator) and how to manipulate
  this concept in grid2op. Redispatching allows you to perform **continuous** actions on the powergrid problem.